resources/clj-kondo.exports/nextjournal/clerk/nextjournal/clerk/viewer.clj_kondo

```diff
@@ -2,11 +2,14 @@
   (:require [clj-kondo.hooks-api :as api]
             [clojure.string :as str]))
 
-(defn ->viewer-fn [{:keys [node]}]
-  (let [[name-node quoted-node] (:children node)
+(defn ->viewer-fn [{:keys [node lang] :as opts}]
+  (let [[_name-node quoted-node] (:children node)
         quoted-tag (api/tag quoted-node)]
     (when (= :quote quoted-tag)
-      (let [quoted-node {:tag :syntax-quote
-                         :children (:children quoted-node)}
-            fn-node (first (:children quoted-node))]
-        {:node fn-node}))))
+      (let [quoted-node
+            (with-meta
+              {:tag :syntax-quote
+               :children (:children quoted-node)}
+              (assoc (meta node)
+                     :clj-kondo/ignore [:unresolved-var]))]
```

Comment:
This is silencing the `:unresolved-var` warning now, correct? Would it also be possible to keep it, but always look it up via `cljs`?

Reply:
This isn't possible, but navigation in lsp still works since we implemented the .cljs fallback in December

Reply:
Ok, merged it. Would you be open to making this possible in hooks? Besides in Clerk we have a few other examples where this would be nice to have.